components/frontend-react/src/containers/LoginComponent.js

```diff
@@ -1,19 +1,52 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
+import axios from 'axios';
 
 import Appbar from '../components/Appbar';
 import BassaIntroBox from '../components/BassaIntroBox';
 import UserSignup from '../components/UserSignup';
 
+const APIURL = "http://0.0.0.0:5000/api";
```

Comment:
Get the URL from .env.

Create .env file and have REACT_APP_API_URL or some env variable in the file. Create React app will autoload .env file and REACT_APP_* variables when we run and build application.

further reading https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables

Reply:
Thanks @iamchathu for your guidance and advice. I will add .env file for that.